system/jlib/jsocket.cpp

```diff
@@ -893,6 +894,8 @@ int CSocket::post_connect ()
         nagling = true;
         set_nagle(false);
         state = ss_open;
+        char lname[256];
+        localPort = name(lname, sizeof(lname), false);
```

Comment:
The oktothrow=false, is it anymore expect to throw an exception here than anywhere else?
The previously (prior to this commit) it would throw because called too early.